src/components/K8s.vue

```diff
@@ -85,17 +145,44 @@ export default {
     },
     handleCheckbox(event, name) {
       ipcRenderer.send('install-set', name, event.target.checked);
-    }
+    },
+    handleUpdateMemory(value) {
+      this.settings.kubernetes.memoryInGB = value;
+      if (this.memoryValueIsValid) {
+        this.debouncedActOnUpdateMemory();
+      }
+    },
+    handleUpdateCPU(value) {
+      this.settings.kubernetes.numberCPUs = value;
+      if (this.numCPUsValueIsValid) {
```

Comment:
If it's somehow _not_ valid, we should be showing a message somewhere; it's somewhat confusing for the machine to take my input and then do nothing with it (which I could observe when I set it to 1 CPU).

Reply:
Good call. This is a holdover from the input field, I need to think about it.

Reply:
I… had a slider from 1 to (number of CPUs in the host), I move the slider to 1, and watch the config file on disk and see that no changes were applied?  I might be on an older revision of this PR, though.

Reply:
Done. A little overkill now, but we do need a strategy for communicating to the user.